src/queries/index.ts

```diff
@@ -0,0 +1,10 @@
+import {Query} from 'compassql/build/src/query/query';
+import {Store} from 'redux';
+import {StateWithHistory} from 'redux-undo';
+import {StateBase} from '../../build/src/models/index';
+import {resultRequest} from '../actions/result';
+
+export function makeQueries(store: Store<StateWithHistory<Readonly<StateBase>>>, query: Query) {
```

Comment:
the verb 'make' is consistent with other areas in the code where we dispatch ? or is this the first time this is happening?